python-crud-google-spreadsheet/src/create.py

```diff
@@ -0,0 +1,18 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+import os
+
+
+def handler(body, req):
+    if (body is None or body['values'] is None or
```

Comment:
I think you might be missing some edge cases here, for example - an empty list. Consider using a JSON schema for input validation.